README.md

```diff
@@ -43,7 +43,13 @@ _Coming soon_
 
 THAT website is server-side rendered React via Next. To get started, clone the repo and run `npm install` to get all the dependencies in place. Also run `npm i -g now` to install [Zeit Now](https://zeit.co/docs) globally on your machine.
 
-Run `cp .env.sample .env && cp .env.build.sample .env.build` to get basic configuration in place, and then `now dev` to startup `localhost`.
+Run `cp .env.sample .env && cp .env.build.sample .env.build` to get basic configuration in place.
+
+Edit the .env.build file. `SESSION_COOKIE_SECRET` should be any 36 character value. `API_GATEWAY` should be `https://api.that.tech`
```

Comment:
Cookie secret requires at least 32 character value.

Reply:
Got it! Updated.